app/models/post.rb

```diff
@@ -0,0 +1,43 @@
+# A post is a message viewable by members if published. The author is polymorphic however
+# for now only use from an admin user is possible
+class Post < ApplicationRecord
+  include Sidekiq::Worker
+
+  belongs_to :author, :polymorphic => true
+  serialize :tags
+
+  validates :status, presence: true
+  validates :published_at, presence: true
+
+  enum status: [:draft, :published, :review]
+
+  default_scope { where.not(published_at: nil).order(published_at: :desc) }
+  scope :pinned, -> { where(id: Settings['posts.pinned']) }
+  scope :unpinned, -> { where.not(id: Settings['posts.pinned']) }
+
+  attr_accessor :pinned
+
+  def tags=(tags)
+    write_attribute(:tags, tags.split(' '))
+  end
+
+  def pinned?
+    Settings['posts.pinned'].include? id
```

Comment:
Should this not just be a boolean in the database instead of storing this in the settings?
This would simplify the code and prevent logic errors

Reply:
Fully agree, will fix this soontm

Reply:
Has been fixed